linked_list/linkedList.js

```diff
@@ -41,11 +41,11 @@ class LinkedList {
 
 	// Adds value at end of Linked List
 	addLast (value) {
-		if(this.head === null) {
+		if (this.head === null) {
 			this.addFirst(value);
 		} else {
 			let temp = this.head;
-			while(temp.next != null) {
+			while (temp.next != null) {
```

Comment:
Good going @aashutoshrathi , since you used strict equality above so i thought it would be better to keep it homogeneous.